team_delta.js

```diff
@@ -1 +1,39 @@
 // Fetch all routes for the airline and insert into #delta
+
+const getFlightLisNyc = async () => {
```

Comment:
`getFlightLisNyc`, `getFlightTlvBer` and `getFlightArnLhr` all kind of do the same thing, maybe merge them into one function?